medicines/doc-index-updater/Cargo.toml

```diff
@@ -1,20 +1,24 @@
 [package]
 name = "doc_index_updater"
 version = "0.1.0"
-authors = ["Stuart Harris <stuart.harris@red-badger.com>", "Robin James Kerrison <robinjames.kerrison@red-badger.com>", "Matt Doughty <matt.doughty@red-badger.com>"]
+authors = ["Stuart Harris <stuart.harris@red-badger.com>", "Robin James Kerrison <robinjames.kerrison@red-badger.com>", "Matt Doughty <matt.doughty@red-badger.com>", "Pedro Martin <pedro.martin@red-badger.com>"]
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+azure_sdk_service_bus = { git = "https://github.com/redbadger/AzureSDKForRust", branch = "service-bus-read-event" }
+azure_sdk_core = { git = "https://github.com/redbadger/AzureSDKForRust", branch = "service-bus-read-event" }
```

Comment:
We can update these to reference the original repo now that your PR was merged!

Reply:
On the newest version of [azure_sdk_service_bus](https://crates.io/crates/azure_sdk_service_bus)  and [azure_sdk_core](https://crates.io/crates/azure_sdk_core) are some breakinhg changes.

Let's keep this until we sort it out